lib/buildFactory.js

```diff
@@ -226,6 +229,12 @@ class BuildFactory extends BaseFactory {
                         return super.create(modelConfig);
                     });
                 })
+                    .then(build => Promise.all(modelConfig.steps.map(step =>
```

Comment:
do we still populate `steps` field for build model ? Is that required ?

Reply:
don't really need it. i can delete it from modelConfig before we save the build.

Reply:
I think it would be good to leave it just in case.